[2024-02-13T17:05:03.125+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: deverlop_model.data_sets scheduled__2024-02-13T17:00:00+00:00 [queued]>
[2024-02-13T17:05:03.134+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: deverlop_model.data_sets scheduled__2024-02-13T17:00:00+00:00 [queued]>
[2024-02-13T17:05:03.134+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2024-02-13T17:05:03.145+0000] {taskinstance.py:2191} INFO - Executing <Task(DockerOperator): data_sets> on 2024-02-13 17:00:00+00:00
[2024-02-13T17:05:03.151+0000] {standard_task_runner.py:60} INFO - Started process 1467 to run task
[2024-02-13T17:05:03.155+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'deverlop_model', 'data_sets', 'scheduled__2024-02-13T17:00:00+00:00', '--job-id', '200', '--raw', '--subdir', 'DAGS_FOLDER/task.py', '--cfg-path', '/tmp/tmpkndmmrm_']
[2024-02-13T17:05:03.157+0000] {standard_task_runner.py:88} INFO - Job 200: Subtask data_sets
[2024-02-13T17:05:03.193+0000] {task_command.py:423} INFO - Running <TaskInstance: deverlop_model.data_sets scheduled__2024-02-13T17:00:00+00:00 [running]> on host a5e5e25b3882
[2024-02-13T17:05:03.256+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='rtn' AIRFLOW_CTX_DAG_ID='deverlop_model' AIRFLOW_CTX_TASK_ID='data_sets' AIRFLOW_CTX_EXECUTION_DATE='2024-02-13T17:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-02-13T17:00:00+00:00'
[2024-02-13T17:05:03.271+0000] {docker.py:359} INFO - Starting docker container from image python:3.8
[2024-02-13T17:05:05.100+0000] {docker.py:429} INFO - WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
[2024-02-13T17:05:15.277+0000] {docker.py:429} INFO - READ DATA :  HELLO WORLD  timestamp :  2024-02-13 17:05:15.273617
[2024-02-13T17:05:15.731+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=deverlop_model, task_id=data_sets, execution_date=20240213T170000, start_date=20240213T170503, end_date=20240213T170515
[2024-02-13T17:05:15.772+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-02-13T17:05:15.792+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
